webpack.config.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'development',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+};
```

Comment:
- [ ] It was required to have an HTML file configured in the Webpack config settings as a template using the HTML Webpack Plugin.